faqs/pt/cdp.md

```diff
@@ -0,0 +1,137 @@
+# Posições de dívida colateralizada (CDP)
+
+## O que são posições de dívida colateralizada (CDP)?
+
+A posição de dívida colateralizada (CDP) do MakerDAO é um tipo de empréstimo gerenciado por um sistema de contrato inteligente executado na blockchain Ethereum. O CDP é um componente fundamental do Sistema Stablecoin Dai, que permite a criação do Dai contra as garantias depositadas que são retidas até o retorno do Dai emprestado.
+
+O uso do CDP altera coletivamente a oferta total de Dai. Os usuários criam o Dai emprestando-o contra suas garantias e o Dai é destruído quando eles pagam sua dívida. Esse processo ocorre na blockchain, que permite auditoria completa e independente do Dai e das garantias que o suportam.
+
+O CDP é obrigado a ter um excesso de garantias. É necessário que o valor colateral de um CDP seja maior que o valor da dívida, o que garante aos usuários da Dai que seu Dai tem valor e é totalmente lastreado em ativos reais.
+
+## Como um CDP funciona?
+
+Qualquer usuário que deseje solicitar um empréstimo no Dai pode depositar o ETH em um CDP. Uma vez depositado, o usuário pode gerar Dai contra o valor do seu depósito. O ETH depositado pode ser retirado proporcionalmente quando o usuário retorna parte ou todo o Dai emprestado. Desde que os usuários do CDP mantenham a [Taxa de colateralização] (https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio), eles podem retirar ou depositar livremente o excesso de garantia.
```

Comment:
Links won't display correctly if there is a space between the square and round brackets:

[Taxa de colateralização] (https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio)

[Taxa de colateralização](https://github.com/makerdao/community/blob/master/faqs/cdp.md#what-is-the-collateralization-ratio)

Reply:
These links should be relative as well. 

`[Taxa de colateralização](../cdp.md#what-is-the-collateralization-ratio)`